src/wp-includes/blocks/post-excerpt.php

```diff
@@ -83,7 +83,7 @@ function register_block_core_post_excerpt() {
  * the excerpt length block setting has no effect.
  * Returns 100 because 100 is the max length in the setting.
  */
-if ( is_admin() ||
+if ( is_admin() &&
 	defined( 'REST_REQUEST' ) && REST_REQUEST ) {
 	add_filter(
```

Comment:
This does not work, as `is_admin()` and `REST_REQUEST` are never both true at the same time.

Also, any changes to this file need to be done in Gutenberg: https://github.com/WordPress/gutenberg/blob/d5d8533cf2cc04bb005bda147114cf00782d6c38/packages/block-library/src/post-excerpt/index.php#L80-L95